src/Og.php

```diff
@@ -169,6 +171,31 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
   }
 
   /**
+   * Return TRUE if entity belongs to a group.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $group
+   *   The group entity to check.
+   * @param \Drupal\Core\Entity\EntityInterface $entity
+   *   The entity to get groups for.
+   * @param array $states
+   *   (optional) Array with the membership states to check the membership.
+   *   Defaults to active memberships.
+   *
+   * @return bool
+   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
+   *   or blocked.
+   */
+  public static function isMember(EntityInterface $group, EntityInterface $entity, $states = [OG_STATE_ACTIVE]) {
```

Comment:
I think we were going to review this after.  But just moving stuff into
services and using those services instead.
On 7 Dec 2015 18:02, "Amitai Burstein" <notifications@github.com> wrote:

> In src/Og.php
> <https://github.com/amitaibu/og/pull/53#discussion_r46853442>:
>
> > @@ -169,6 +171,31 @@ public static function getEntityGroups($entity_type, $entity_id, $states = [OG_S
> >    }
> >
> >    /**
> > +   * Return TRUE if entity belongs to a group.
> > +   *
> > +   * @param \Drupal\Core\Entity\EntityInterface $group
> > +   *   The group entity to check.
> > +   * @param \Drupal\Core\Entity\EntityInterface $entity
> > +   *   The entity to get groups for.
> > +   * @param array $states
> > +   *   (optional) Array with the membership states to check the membership.
> > +   *   Defaults to active memberships.
> > +   *
> > +   * @return bool
> > +   *   TRUE if the entity (e.g. the user) belongs to a group and is not pending
> > +   *   or blocked.
> > +   */
> > +  public static function isMember(EntityInterface $group, EntityInterface $entity, $states = [OG_STATE_ACTIVE]) {
>
> hi @dawehner <https://github.com/dawehner> welcome to the party! :)
>
> Do we really need static methods?
>
> What would be the alternative here? Or in other words, why not static?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/amitaibu/og/pull/53/files#r46853442>.
>